src/MyPlot/MyPlot.php

```diff
@@ -1114,23 +1187,24 @@ public function onLoad() : void {
 		$this->getServer()->getCommandMap()->register("myplot", new Commands($this));
 	}
 
-	public function onEnable() : void {
+	public function onEnable() : void
+	{
 		if (!\class_exists(SpoonDetector::class)) {
 			$this->getServer()->getPluginManager()->disablePlugin($this);
 			return;
 		}
 		SpoonDetector::printSpoon($this, "spoon.txt");
-		if($this->isDisabled()) {
+		if ($this->isDisabled()) {
 			return;
 		}
 		$this->getLogger()->debug(TF::BOLD . "Loading Events");
 		$eventListener = new EventListener($this);
 		$this->getServer()->getPluginManager()->registerEvents($eventListener, $this);
 		$this->getLogger()->debug(TF::BOLD . "Registering Loaded Levels");
-		foreach($this->getServer()->getLevels() as $level) {
+		foreach ($this->getServer()->getLevels() as $level) {
 			$eventListener->onLevelLoad(new LevelLoadEvent($level));
 		}
-		$this->getLogger()->debug(TF::BOLD.TF::GREEN."Enabled!");
+		$this->getLogger()->debug(TF::BOLD . TF::GREEN . "Enabled!");
```

Comment:
formatting

Reply:
Your string concatenating is not consistent. Sometimes there are spaces and sometimes not.